### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by mdmollah**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+			<groupId>com.squareup.okhttp</groupId>
+			<artifactId>okhttp</artifactId>
+			<version>2.7.5</version>
+		</dependency>
+		<dependency>
+			<groupId>com.squareup.okhttp</groupId>
+			<artifactId>logging-interceptor</artifactId>
+			<version>2.7.5</version>
+		</dependency>
+		<dependency>
```
Comment (yinchanted) [inline pom.xml]: Which version of "okhttp" are we using? 2.7.5 or 3.10.0?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+			<version>1.7.1</version>
+		</dependency>
+		<dependency>
+			<groupId>io.swagger</groupId>
+			<artifactId>swagger-annotations</artifactId>
+			<version>1.5.20</version>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.mockito/mockito-all -->
+		<dependency>
+			<groupId>org.mockito</groupId>
```
Comment (yinchanted) [inline pom.xml]: Duplicated entry.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>com.swift</groupId>
+		<artifactId>parent-pom</artifactId>
+		<version>1.0.0</version>
+	</parent>
+
+	<groupId>com.swift.api.gpi</groupId>
+	<artifactId>tracker-v2</artifactId>
```
Comment (yinchanted) [inline pom.xml]: change to trackerv2

File: pom.xml (hunk 2 above)
Comment (mdmollah) [reply pom.xml]: added trackerv2 in groupid and remove duplicate dependecy from pom file

File: pom.xml (hunk 3 above)
Comment (mdmollah) [reply pom.xml]: done

File: pom.xml (hunk 1 above)
Comment (mdmollah) [reply pom.xml]: latest one 3.10.0

**Pull request merged**